### ISSUE TITLE
Build Failed - missing required architecture arm64 

### ISSUE THREAD

**Issue by yidan** [steventroughtonsmith/tvOSBrowser#24]
ld: in 

/System/Library/PrivateFrameworks/UIFoundation.framework/UIFoundation', missing required architecture arm64 in file
/System/Library/PrivateFrameworks/UIFoundation.framework/UIFoundation (2 slices) for architecture arm64 clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by yidan** [comment]
The application build failed after making an upgrade to Xcode 9.

**Comment by hadihashem** [comment]
Hi @yidan 
I'm facing the same issue
Did you already solve it?
Regards

**Comment by yidan** [comment]
@hadihashem Yes! Open a new project in Xcode and copy the code over, instead of cloning the git repo!
But now I'm facing another issue. When I change the URL, the webview does not go into full screen. Let me know if it happens to you as well.

**Comment by hadihashem** [comment]
Thank you for the tip
My problem is solved
I also noticed, only for some URLs, a problem to go into full screen (the webpage is shifted a little bit to the right)
But I don't have a good knowledge in Objective-C to solve this
![2018-08-13_212512](https://user-images.githubusercontent.com/34154847/44053559-53240fbe-9f40-11e8-8399-9fab416081dc.jpg)

**Comment by yidan** [comment]
@hadihashem I found a hack! Replace 
self.webview = [[UIWebView alloc] initWithFrame:[UIScreen mainScreen].bounds]; 
with 
self.webview = [[UIWebView alloc] initWithFrame:CGRectMake(-100, -80, 2120, 1260)];

This will shift the screen to the left, hiding the grey border. Not a perfect solution, but it works fine for my use case.

**Comment by hadihashem** [comment]
@yidan It works also for me 
I only had to adjust the pixels
Thank you :-)
